adafruit_displayio_layout/widgets/cartesian.py

```diff
@@ -247,6 +247,24 @@ def __init__(
         self._screen_palette[4] = 0xFFFFFF
         self._screen_palette[5] = self._background_color
 
+        self._corner_bitmap = displayio.Bitmap(10, 10, 5)
```

Comment:
@FoamyGuy Here is the inclusion to solve the corner issue, I tested with differents strokes, however a second pair of eyes wuld not hurt